xml/System.Net.Http/HttpClient.xml

```diff
@@ -1124,7 +1124,7 @@ public class GoodController : ApiController
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This operation will not block. The returned <xref:System.Threading.Tasks.Task%601> object will complete after the whole response body is read. This method does not buffer the stream.  
+ This operation will not block. The returned <xref:System.Threading.Tasks.Task%601> object will complete after the response headers are read. This method does not read not buffer the response body.  
```

Comment:
"read not" -> "read nor"

Reply:
Sorry about the typo, @davidsh. I've corrected it.